trading_calendars/trading_calendar.py

```diff
@@ -812,20 +920,16 @@ def all_minutes(self):
         """
         Returns a DatetimeIndex representing all the minutes in this calendar.
         """
-        opens_in_ns = self._opens.values.astype(
-            'datetime64[ns]',
-        ).view('int64')
-
-        closes_in_ns = self._closes.values.astype(
-            'datetime64[ns]',
-        ).view('int64')
-
         return DatetimeIndex(
-            compute_all_minutes(opens_in_ns, closes_in_ns),
+            compute_all_minutes(
+                self.market_opens_nanos,
+                self.market_break_starts_nanos,
+                self.market_break_ends_nanos,
+                self.market_closes_nanos,
+            ),
             tz=UTC,
         )
 
-    @preprocess(dt=coerce(pd.Timestamp, attrgetter('value')))
```

Comment:
If we remove the preprocessor, then do we need to check whether `dt` is a `Timestamp`, since it could be an int?
```
        Parameters
        ----------
        dt : pd.Timestamp or nanosecond offset
            The dt for which to get the containing session.
```

If we do remove this lone `preprocess` call site, then I think we can actually kill the entire preprocess module.